.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +1,31 @@
-## What changes were proposed in this pull request?
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If this is your first time, please read our contributor guidelines: https://spark.apache.org/contributing.html
+2. Ensure you have added or ran the appropriate tests for your PR: https://spark.apache.org/developer-tools.html
+3. If the PR is unfinished, add `[WIP]` in your PR title.
+4. Update your PR description whenever it is needed.
```

Comment:
Also, if possible, I'd like PR authors to put a very simple example query in PR descriptions to reproduce an issue (Especially for fixes in the SQL part).

Reply:
Actually I think that's at `How was this patch tested?`